crates/bevy_render/src/lib.rs

```diff
@@ -294,6 +287,17 @@ impl Plugin for RenderPlugin {
     }
 }
 
+fn try_create_surface(app: &mut App, wgpu_instance: &wgpu::Instance) -> Option<Surface> {
```

Comment:
This should return a `Result`, not an `Option` IMO.

Reply:
I can see why that would make sense. Then the caller would know _why_ they didn't get a surface (no window resource for example). On the other hand, we don't actually use the reason for not getting a surface anywhere (yet?).
If this returned an error, currently we'd have to convert it back into an Option to pass it into `RequestAdapterOptions`

Reply:
Hmm, that's fair. Since this isn't `pub` that's probably fine for now.